.github/workflows/ci.yml

```diff
@@ -28,7 +28,17 @@ jobs:
             cling: Off
             cppyy: On
             llvm_enable_projects: "clang;lld"
```

Comment:
lld not required here. The only case where is required is clang-repl-19 + emscripten.

Reply:
Done.